### PR TITLE
add new parameter node_extra_taints

### PR DESCRIPTION
**Pull request by BaronMsk**
parameter node_extra_taints Additional taints for new instaled nodes.

### CODE REVIEW

**Review by ybolotnikov — APPROVED** [review]

**Comment by GSPatton** [comment]
Hi @BaronMsk. Could you add a description to this PR? Also, this change would need additional tests. Could you add some? 

Thanks

**Comment by BaronMsk** [comment]
@GSPatton i added some tests

File: templates/v1beta3/config_worker.yaml.erb [hunk 1]
```diff
@@ -14,6 +14,19 @@ discovery:
...
+    <%- end -%>
+    <%- if item['operator'] -%>
+    operator: <%= item['operator'] %>
+    <%- end -%>
+    effect: <%= item['effect'] %>
+  <%- end -%>
+<%- end -%>
   <%- if @container_runtime == "cri_containerd" -%>
   criSocket: unix:///run/containerd/containerd.sock
   taints: null
```
Comment (LukasAud) [inline templates/v1beta3/config_worker.yaml.erb:32]: Shouldn't this line be removed the same way it happened in templates/v1beta2/config_worker.yaml.erb ?

Comment (BaronMsk) [reply templates/v1beta3/config_worker.yaml.erb]: done

**Comment by jordanbreen28** [comment]
Hi @BaronMsk - thanks for all your work on this one.
This looks good to merge now, so can you rebase with main and we can proceed.

**Review by jordanbreen28 — APPROVED** [review]
LGTM

**Pull request merged**